public/images/icon-download-black.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20.77 24.39"><defs><style>.cls-1{fill:#090909;fill-rule:evenodd;}</style></defs><title>icon-download-black</title><path class="cls-1" d="M10.87,18.94a.64.64,0,0,1-1,0L2,10.89a.52.52,0,0,1-.22-.44A.87.87,0,0,1,1.89,10L3,8.87a.75.75,0,0,1,.48-.16.49.49,0,0,1,.43.22L9,14.21V.65A.62.62,0,0,1,9.17.19.6.6,0,0,1,9.63,0h1.51a.6.6,0,0,1,.46.19.62.62,0,0,1,.19.46V14.21l5.09-5.28a.49.49,0,0,1,.43-.22.8.8,0,0,1,.49.16l1,1.09a.67.67,0,0,1,.22.49.6.6,0,0,1-.16.44Zm9.25,2.62a.62.62,0,0,1,.46.19.6.6,0,0,1,.19.46v1.52a.63.63,0,0,1-.19.47.66.66,0,0,1-.46.19H.65a.66.66,0,0,1-.46-.19A.63.63,0,0,1,0,23.73V22.21a.6.6,0,0,1,.19-.46.62.62,0,0,1,.46-.19Z"/></svg>
```

Comment:
We have an [svg sprite](https://github.com/greenpeace/planet4-master-theme/blob/develop/images/symbol/svg/sprite.symbol.svg) on master-theme, that already includes share icons. Can you use these instead?